rs/tests/Cargo.toml

```diff
@@ -590,6 +590,10 @@ path = "testing_verification/testnets/large.rs"
 name = "ic-systest-consensus-performance"
 path = "consensus/consensus_performance.rs"
 
+[[bin]]
+name = "ic-systest-adding-nodes-to-subnet-test"
+path = "consensus/adding_nodes_to_subnet_test.rs"
+
```

Comment:
What about moving this to a `Cargo.toml` to `rs/tests/consensus`?

Reply:
Good idea, let me take a look

Reply:
PTAL @basvandijk